### PR TITLE
Adding fhir indexer project to communities

### PR DESCRIPTION
**Pull request by royashcenazi**

### CODE REVIEW

File: docs/ecosystem/community/fhir-indexer.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+id: fhir-indexer
+title: "Fhir-indexer"
+---
+
+[Fhir-indexer](https://github.com/royashcenazi/fhir-indexer) is a ZIO based library for fetchine FHIR resources fast and easy.
```
Comment (guizmaii) [inline docs/ecosystem/community/fhir-indexer.md]:
```suggestion
[Fhir-indexer](https://github.com/royashcenazi/fhir-indexer) is a ZIO based library for fetching FHIR resources fast and easy.
```

**Review by adamgfraser — APPROVED** [review]

**Comment by royashcenazi** [comment]
Hey @adamgfraser,
Can you please check why this change not in UI yet? could be that changes are not released still?

**Comment by adamgfraser** [comment]
@khajavi Can you take a look?

**Comment by khajavi** [comment]
@royashcenazi 

> Can you please check why this change not in UI yet? could be that changes are not released still?

Because you haven't updated the `website/sidebars.json` file to point to the newly added page.

**Comment by khajavi** [comment]
~~Update: The `zio-ecosystem-docusaurus` plugin needs to be changed, to automatically get all metadata from `package.json` and `sidebars.json` files. With this change, I hope this issue will be fixed.~~

**Pull request merged**